docs/UBUNTU24_DESKTOP_VM_SETUP.md

```diff
@@ -184,16 +197,11 @@ $ composer update
 $ composer install
 $ composer setup
 $ sail up -d
-$ sail root-shell
- # cd ..
- # chown -R sail:sail html
- # exit
 $ sail artisan ra:storage:link
-$ php composer.phar install
+$ rm public/fi-build
```

Comment:
`sail artisan ra:storage:link` creates this folder, but with different permissions than whatever uses it needs. By removing the folder, the other process can create and populate is appropriately.